src/BusinessLayer/TelemetryReporting.cpp

```diff
@@ -142,7 +153,7 @@ void TelemetryReporting::sendCmuData(unsigned char cmuDataIndex)
    sendData(packet, packetLength);
 }
 
-void TelemetryReporting::sendMpptData(unsigned char mpptDataIndex)
+void TelemetryReporting::sendMppt(unsigned char mpptDataIndex)

```

Comment:
What's the motivation for changing the names of this function and `sendCmuData()`? I feel like the old names were better due to being more descriptive.